structures/heap/heap.tex

```diff
@@ -1 +1,25 @@
-\section{Heaps}
+\section{Heaps}\index{heap}\index{priority\_queue}
+Heaps are very useful data structures that support at least the following operations:
+
```

Comment:
This is not a very comprehensive explanation of heaps, lacking more time complexity information, what they are particularly good at, or other useful details. That said, it is better than nothing.